charts/datadog/templates/_helpers.tpl

```diff
@@ -739,3 +739,15 @@ In 7.36, `--config` was deprecated and `--cfgpath` should be used instead.
 --config
 {{- end -}}
 {{- end -}}
+
+{{/*
+Returns whether or not the underlying OS is Google Container-Optimized-OS
+Note: GKE Autopilot clusters only use COS (see https://cloud.google.com/kubernetes-engine/docs/concepts/node-images)
+*/}}
+{{- define "is-COS" -}}
```

Comment:
I would prefer another name for the function, because here it also check the autopilot option.

maybe something like `can-mount-host-usr-src`

Reply:
agreed, that way if we ever have another env where this is a problem we can update the logic here.
The user facing `gke.cos` is great, here it's more of an internal.